OWML.Common/IModInteraction.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public interface IModInteraction
```

Comment:
Just a suggestion: instead of all this...
`var value = ModHelper.Interaction.GetVariableValue("foo", "bar");
ModHelper.Interaction.InvokeMethod("lolol");`

... how about this?
`var mod = ModHelper.Interaction.GetMod("foo");
var value = mod.GetVariableValue("bar");
mod.InvokeMethod("lolol");`

Reply:
so what would GetMod return? the modhelper?